src/avro.erl

```diff
@@ -99,6 +99,9 @@
              , union_type/0
              ]).
 
+-export([ crc64_fingerprint/1
```

Comment:
move it up to line ~60.
also, add `-spec crc64_fingerprint() :: avro_fingerprint:crc64().` and export it.

Reply:
I moved the export. There is no avro_fingerprint:crc64/0 function, though. Do you mean to export the type?